app/frontend/src/components/Chat/ChannelChatComponent.tsx

```diff
@@ -200,56 +222,79 @@ export default function ChannelChatComponent(props: ChannelChatComponentProps) {
           console.error("Fetch error:", err);
         }
       }
-    } finally {setLoading(false);}
+    } finally {
+      setLoading(false);
+    }
   };
 
-  const removeReaction = (emoji: string, reactions: string[], reactionUsers: IUserModel[]) => {
+  const removeReaction = (
+    emoji: string,
+    reactions: string[],
+    reactionUsers: IUserModel[],
+  ) => {
     let newReactions = [];
-    for(let i = 0; i < reactions.length; i++) if(!(reactions[i] === emoji && reactionUsers[i].user_id == (userStore.user ? userStore.user.user_id : -2))) newReactions.push(reactions[i]);
+    for (let i = 0; i < reactions.length; i++)
+      if (
+        !(
+          reactions[i] === emoji &&
+          reactionUsers[i].user_id ==
+            (userStore.user ? userStore.user.user_id : -2)
+        )
+      )
+        newReactions.push(reactions[i]);
     return newReactions;
-  }
-  const removeUser = (emoji: string, reactions: string[], reactionUsers: IUserModel[]) => {
+  };
+  const removeUser = (
+    emoji: string,
+    reactions: string[],
+    reactionUsers: IUserModel[],
+  ) => {
     let newUsers = [];
-    for(let i = 0; i < reactions.length; i++) if(!(reactions[i] === emoji && reactionUsers[i].user_id == (userStore.user ? userStore.user.user_id : -2))) newUsers.push(reactionUsers[i].user_id);
+    for (let i = 0; i < reactions.length; i++)
+      if (
+        !(
+          reactions[i] === emoji &&
+          reactionUsers[i].user_id ==
+            (userStore.user ? userStore.user.user_id : -2)
+        )
+      )
+        newUsers.push(reactionUsers[i].user_id);
     return newUsers;
-  }
+  };
 
   const sendMessage = () => {
-  
-
     const finalMessage = audioURL ? "AUDIO" : message.trim();
 
     if (!finalMessage) {
-      return; 
+      return;
     }
 
     let replyInfo = null;
     if (replyingTo !== null) {
       const repliedMessage = messages[replyingTo];
       if (repliedMessage) {
         replyInfo = {
-          replyToId: replyingTo, 
+          replyToId: replyingTo,
           replyToUsername: repliedMessage.username,
           replyToMessage: repliedMessage.message,
         };
       }
     }
 
-
     ChannelChatService.sendMessageToChannel(
       props.channelId,
       props.userId,
       finalMessage,
       replyInfo,
-      audioURL 
+      audioURL,
     );
 
     setMessage("");
     setAudioBlob(null);
     setAudioURL(undefined);
-    setReplyingTo(null); 
+    setReplyingTo(null);
     activitySubmit(UserActivity.Online);
-};
+  };
```

Comment:
Add this here and also in DMChatComponent - we want the chatbar to scroll down on message send by current user in all circumstances - whether they're already at the bottom or not.

```suggestion
    scrollToBottom();
  };
```

Reply:
We tested some more and this wasn't the right fix, instead we opted to utilize the useEffect that was implemented.